runtime/src/main/java/io/micronaut/health/HealthStatus.java

```diff
@@ -29,7 +28,6 @@
  * @author Graeme Rocher
  * @since 1.0
  */
-@Immutable
```

Comment:
Why was this removed?

Reply:
`@Immutable` is inside the the findbugs jar file which we can no longer use. The alternatively is to leave this annotation there and make find bugs `compileOnly`